services/api-server/src/simcore_service_api_server/models/schemas/model_adapter.py

```diff
@@ -121,3 +125,20 @@ class ServicePricingPlanGetLegacy(BaseModel):
 assert set(ServicePricingPlanGetLegacy.model_fields.keys()) == set(
     _ServicePricingPlanGet.model_fields.keys()
 )
+
+
+class LicensedItemGet(BaseModel):
+    model_config = ConfigDict(
```

Comment:
Everywhere else (including the pydantic manuals) , we have the `model_config` *after* the field names ... :-)

Reply:
fixed